### ISSUE TITLE
Typos in template pools and nbt files preventing some village pieces from spawning

### ISSUE THREAD

**Issue by V-Boy** [TelepathicGrunt/RepurposedStructures#29]
https://hatebin.com/zmwdsvspov

**Comment by V-Boy** [comment]
https://hatebin.com/nqdopanckf

I think they might all be broken in general

**Comment by TelepathicGrunt** [comment]
Are you using Blame 1.4.3? And do you have any datapacks on?
And what mods do you have on? 

I ask cause Repurposed Structures was working for me moments ago while testing with another mod so your list of mods might had broken something. If you can get me the entire latest.log file with latest blame on, that would be helpful too!

**Comment by V-Boy** [comment]
[latest.log](https://github.com/TelepathicGrunt/RepurposedStructures/files/5381599/latest.log)
Yea, i'm using 1.4.3

**Comment by V-Boy** [comment]
No datapacks either, but yeah I kinda have a hefty pack. I can do some binary searching to see whats causing to pop it up, not sure where to start though.

**Comment by TelepathicGrunt** [comment]
Is there a place I can download the pack from or for you to export it to me? I can do a binary search to to help. I tend to just cut out the top half of mods, see if issue persist, then cut the problematic side in half and so on.

**Comment by V-Boy** [comment]
Sure, yeah here.

http://www.mediafire.com/file/3ma149mcn5d3w4w/mods.zip/file

**Comment by V-Boy** [comment]
https://hatebin.com/nyyhptfchc
Teleporting to a badlands village will prompt this up sometimes, odd

**Comment by V-Boy** [comment]
I can confirm that the main structures being affected are giant taiga, jungle, mountain, and swamp villages. Can't really confirm if it happens to nether villages since i haven't gone there yet.

**Comment by V-Boy** [comment]
[debug.log](https://github.com/TelepathicGrunt/RepurposedStructures/files/5395047/debug.log)
[latest.log](https://github.com/TelepathicGrunt/RepurposedStructures/files/5395048/latest.log)
I don't think its another mod, I just ran up the game with just Blame, RS, and the latest forge and my log still got the blame reports.
![2020-10-17_00 56 47](https://user-images.githubusercontent.com/40730042/96329580-01c4d800-1014-11eb-90b2-e6eaefdf90ec.png)

**Comment by TelepathicGrunt** [comment]
Oh I see. When I tested, I just went and opened a world but didn't go to structures. I took a closer look and oh my god. I think there's typos in the template pools on my end lol. How did I miss that?! 

Anyway, thank you for telling me! I'll teleport to every structure now and fix the typos so those nbt files spawn again. The village is generating all the other pieces except for those that Blame found is not correct. Like this one:
`repurposed_structures:structures/village/giant_taiga/houses/butcher_shop_2.nbt` should be
`repurposed_structures:structures/village/giant_taiga/houses/butchers_shop_2.nbt`

My bad. Sorry about that!
